src/main/java/de/tum/in/www1/exerciseapp/web/rest/ResultResource.java

```diff
@@ -176,9 +176,10 @@ public ResultResource(ResultRepository resultRepository, Optional<LtiService> lt
     @PreAuthorize("hasAnyRole('USER', 'TA', 'INSTRUCTOR', 'ADMIN')")
     @Timed
     public ResponseEntity<List<Result>> getResultsForParticipation(@PathVariable Long courseId,
-                                                   @PathVariable Long exerciseId,
-                                                   @PathVariable Long participationId,
-                                                   @RequestParam(defaultValue = "true") boolean showAllResults) {
+                                                                   @PathVariable Long exerciseId,
+                                                                   @PathVariable Long participationId,
+                                                                   @RequestParam(defaultValue = "true") boolean showAllResults,
+                                                                   @RequestParam(defaultValue = "false") boolean ratedOnly) {
```

Comment:
Did you add this flag for other exercises (e.g. programming) as well?

Reply:
In the front-end code, where this Endpoint is used, the flag is set to true for QuizExercises and false for all other types of exercises.